bottlecap/Cargo.toml

```diff
@@ -8,16 +8,20 @@ edition = "2021"
 [dependencies]
 figment = { version = "0.10.15", features = ["yaml", "env"] }
 log = "0.4.21"
-tokio = "1.37.0"
 serde = { version = "1.0.197", features = ["derive"], default-features = false }
-ureq = { version = "2.9.6", features = ["json"], default-features = false }
-
+ureq = { version = "2.9.6", features = ["json", "tls"], default-features = false }
 tracing = {version = "0.1.40", default-features = false }
 tracing-subscriber = { version = "0.3.18", default-features = false, features = ["fmt", "env-filter", "json"] }
+ustr = "1.0.0"
+thiserror = "1.0.58"
+fnv = "1.0.7"
+hashbrown = "0.14.3"
+serde_json = "1.0.116"
```

Comment:
One suggestion would be to stick with the more explicit pattern of specifying features et al above. It's annoying but you won't be surprised when you do a version upgrade and new features/deps get flagged in.

Reply:
oh good point. Done.